percona-packaging/scripts/psmdb_builder.sh

```diff
@@ -0,0 +1,783 @@
+#!/bin/sh
+
+shell_quote_string() {
+  echo "$1" | sed -e 's,\([^a-zA-Z0-9/_.=-]\),\\\1,g'
+}
+
+usage () {
+    cat <<EOF
+Usage: $0 [OPTIONS]
+    The following options may be given :
+        --builddir=DIR      Absolute path to the dir where all actions will be performed
+        --get_sources       Source will be downloaded from github
+        --build_src_rpm     If it is set - src rpm will be built
+        --build_source_deb  If it is set - source deb package will be built
+        --build_rpm         If it is set - rpm will be built
+        --build_deb         If it is set - deb will be built
+        --build_tarball     If it is set - tarball will be built
+        --install_deps      Install build dependencies(root privilages are required)
+        --branch            Branch for build
+        --repo              Repo for build
+        --psm_ver           PSM_VER(mandatory)
+        --psm_release       PSM_RELEASE(mandatory)
```

Comment:
It would be good to have some examples for --psm_ver and --psm_release

Reply:
I have added the README which includes all possible way to run this script to get all needed staff